sdk/react/panels/cast/CastDeviceListItem.js

```diff
@@ -0,0 +1,82 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Text,
+  View,
+  TouchableHighlight,
+} from 'react-native';
+
+import {
+  BUTTON_NAMES,
+} from '../../constants';
+import castDevicesStyles from '../style/CastDevicesStyles.json';
+import Utils from '../../utils';
+
+const styles = Utils.getStyles(castDevicesStyles);
+const castButtonSize = 35;
+
+export default class DeviceListItem extends PureComponent {
```

Comment:
Pure component rerenders only if props or state change, but it compares only **one level deep**, in other words it makes **shallow** comparison instead of deep. Please check that this component doesn't have **deep** props or state fields that can be changed during the component lifecycle.

Reply:
To simplify I suggest not to use `PureComponent` at this moment.

Reply:
Okay